dist/tools/commit-msg/check.sh

```diff
@@ -6,9 +6,34 @@
 # General Public License v2.1. See the file LICENSE in the top level
 # directory for more details.
 
+. "$(dirname "$0")/../ci/github_annotate.sh"
+
+github_annotate_setup
+
 MSG_MAX_LENGTH=50
 MSG_STRETCH_LENGTH=72
 
+if tput colors &> /dev/null && [ $(tput colors) -ge 8 ]; then
```

Comment:
It seems something in here makes me print 256 first...

Reply:
Fixed. Also colors for POSIX `echo`.